src/BinSkim.Driver/RuleTelemetryLogger.cs

```diff
@@ -48,13 +48,19 @@ internal sealed class RuleTelemetryLogger : IAnalysisLogger
         /// </summary>
         private IOperationHolder<RequestTelemetry>? analysisOperationHolder;
 
+        public FileRegionsCache FileRegionsCache
+        {
+            get; set;
+        }
+
         /// <summary>
         /// Construct a new <see cref="RuleTelemetryLogger"/>.
         /// </summary>
         /// <param name="telemetryClient">The Application Insights telemetry client.</param>
         public RuleTelemetryLogger(TelemetryClient telemetryClient)
         {
             this.telemetryClient = telemetryClient ?? throw new ArgumentNullException(nameof(telemetryClient));
+            this.FileRegionsCache = new FileRegionsCache();
```

Comment:
>FileRegionsCache [](http://example.com/codeflow?start=17&length=16)

this is required to allow the #nullable feature in this file. I will clean this up soon.